docs/source/understanding.rst

```diff
@@ -10,4 +10,5 @@ also of considerations for deploying Calico at large scale.
    l2-interconnectFabric
    arch-felix-and-acl
    calico-neutron-api
-   api-proposal
+   etcd-architecture
```

Comment:
That's a question, I think: do we plan to merge this immediately and move it later, or just hold off until the switch?